core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -51,6 +51,7 @@ public class TaskManagerConfig
     private DataSize maxPartialAggregationMemoryUsage = DataSize.of(16, Unit.MEGABYTE);
     private DataSize maxPartialTopNMemory = DataSize.of(16, Unit.MEGABYTE);
     private DataSize maxLocalExchangeBufferSize = DataSize.of(32, Unit.MEGABYTE);
+    private DataSize maxLocalExchangeBufferSizeForWrite = DataSize.of(128, Unit.MEGABYTE);
```

Comment:
What's the drawback of just tuning the existing `maxLocalExchangeBufferSize` config higher ?